packages/core/src/utils/hasTracingEnabled.ts

```diff
@@ -0,0 +1,23 @@
+import type { Options } from '@sentry/types';
+
+import { getCurrentHub } from '../hub';
+
+// Treeshakable guard to remove all code related to tracing
+declare const __SENTRY_TRACING__: boolean | undefined;
+
+/**
+ * Determines if tracing is currently enabled.
+ *
+ * Tracing is enabled when at least one of `tracesSampleRate` and `tracesSampler` is defined in the SDK config.
+ */
+export function hasTracingEnabled(
+  maybeOptions?: Pick<Options, 'tracesSampleRate' | 'tracesSampler' | 'enableTracing'> | undefined,
+): boolean {
+  if (typeof __SENTRY_TRACING__ === 'boolean' && !__SENTRY_TRACING__) {
```

Comment:
Hmm, I wonder if this is worth it 🤔 WDYT?

Reply:
I think the "downside" is that users that _do not_ configure this will/may still have this code in their bundle, right?